doc/nrf/glossary.rst

```diff
@@ -79,8 +79,8 @@ Glossary
    Devicetree
       A data structure for describing hardware and its boot-time configuration, including peripherals on a board, boot-time clock frequencies, and interrupt lines.
 
-   Devicetree Specification (DTS, DTSpec)
-      A specification that "provides a complete boot program to client program interface definition, combined with minimum system requirements that facilitate the development of a wide variety of systems." Devicetree Specification, Release v0.3.
+   Devicetree Specification (DTSpec)
+      A document that defines the source and binary representations of a devicetree, along with some common characteristics of the data structure, such as interrupts and memory addressing.
```

Comment:
Rationale: the original quote lifted from the spec is really an inheritance from its original definition in [ePAPR](https://elinux.org/images/c/cf/Power_ePAPR_APPROVED_v1.1.pdf) where there was separate boot firmware that ran before the operating system and was responsible for handing off the final DTB to the OS. This is not applicable to Zephyr/NCS.